src/bindings/Dom2.res

```diff
@@ -114,7 +114,15 @@ module Context2d = {
   @send external fillRect: (context2d, int, int, int, int) => unit = "fillRect"
   @send external fill: context2d => unit = "fill"
   @send external getTransform: context2d => domMatrix = "getTransform"
+  @send external moveTo: (context2d, float, float) => unit = "moveTo"
```

Comment:
Although, thinking this through a bit more, since we have anti-aliasing it might cope correctly with floating point values. Would be good to test out.

I've changed my mind, ok to leave as-is for now.